tests/pact/nested-defpacts.repl

```diff
@@ -141,17 +141,17 @@
         )
     )
   ; case 1: all good + module reference
-  (defpact good2:string (m1:module{iface})
-    (step
-      (+ (m1::good) "-nested")
-      )
-    (step
-      (+ (continue (m1::good)) "-nested")
-      )
-    (step
-      (+ (continue (m1::good)) "-nested")
-      )
-  )
+   (defpact good2:string (m1:module{iface})
```

Comment:
Can we undo this? this was probably my editor.

Reply:
Yeah, sure can revert